app/controllers/healthcheck_controller.rb

```diff
@@ -26,4 +21,14 @@ def sendgrid_alive?
   def messagelabs_alive?
     SendgridHelper.smtp_alive?(*PrisonMailer.smtp_settings.values_at(:address, :port))
   end
+
+  def database_active?
+    ActiveRecord::Base.connection.active?
```

Comment:
I think this is the only healthcheck checking the DB that works across all of the apps in MOJ as `ActiveRecord::QueryCache` middleware has been <a href='https://github.com/ministryofjustice/prison-visits/blob/master/config/application.rb#L73'>removed</a>.